### DISCUSSION TITLE
Isotropic PDF plots

### DISCUSSION THREAD

**Discussion by ecoman-geos** [mtex-toolbox/mtex#2214] [Ask Anything]
Hello,

I'm using mtex 5.12.0beta1, and I'm trying to produce isotropic pdf for the 3 axes [100],[010], [001].

In function mtex-5.12.0beta1/SO3Fun/@SO3Fun/plotPDF.m I set the odf weights to the same value as:

numw = size(SO3F.weights,1);
SO3F.weights = sum(SO3F.weights)/numw*ones(numw,1);

For the first 2 axes the plots areperfectly isotropic, while for the [001] one there are some oscillations:

![res300_sbf_pure_A1pt0_lam0pt0_t1pt0_max4pt6_weight=1](https://github.com/user-attachments/assets/e69d6ef7-164f-4aaf-baf6-3ba3569482fa)

any idea why and how to solve this problem?

thanks
Manuele

**Comment by ralfHielscher** [comment]
If you want to have an isotropic ODF simply do
```matlab
cs = crystalSymmetry('432')
SO3F = uniformODF(cs)
```
Ralf.
